src/borg/testsuite/__init__.py

```diff
@@ -2,7 +2,11 @@
 import filecmp
 import functools
 import os
-import posix
+try:
+    import posix # buildin but not everywhere
+except:
```

Comment:
except ImportError or the linters will jump on it.

Reply:
Uhhh yes... Done